src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -49,6 +49,11 @@ export class CalciteValueListItem {
    */
   @Prop() metadata?: object;
 
+  /**
+   * Set this to true to display a remove action that removes the item from the list.
+   */
+  @Prop({ reflect: true }) removable = false;
```

Comment:
Should we make this an optional prop so that users know they don't have to set it?

```
@Prop({ reflect: true }) removable?: boolean;
```

Reply:
+1